spiceaidocs/docs/components/data-connectors/delta-lake.md

```diff
@@ -5,113 +5,142 @@ description: 'Delta Lake Data Connector Documentation'
-- `delta_lake_azure_storage_sas_key`: The shared access signature key for accessing the storage account.
-- `delta_lake_azure_storage_endpoint`: Optional. The endpoint for the Azure Blob storage account.
+| Parameter Name                           | Description                                                            |
+| ---------------------------------------- | ---------------------------------------------------------------------- |
+| `delta_lake_azure_storage_account_name`  | The Azure Storage account name.                                        |
+| `delta_lake_azure_storage_account_key`   | The Azure Storage master key for accessing the storage account.        |
+| `delta_lake_azure_storage_client_id`     | The service principal client id for accessing the storage account.     |
+| `delta_lake_azure_storage_client_secret` | The service principal client secret for accessing the storage account. |
+| `delta_lake_azure_storage_sas_key`       | The shared access signature key for accessing the storage account.     |
+| `delta_lake_azure_storage_endpoint`      | Optional. The endpoint for the Azure Blob storage account.             |
+
+### Google Storage (GCS)
+
+| Parameter Name           | Description                                                  |
+| ------------------------ | ------------------------------------------------------------ |
+| `google_service_account` | Filesystem path to the Google service account JSON key file. |
+
+## Examples
+
+### Delta Lake + Local
+
+```yaml
+  - from: delta_lake:/path/to/local/delta/table  # A local filesystem path to a Delta Lake table
+    name: my_delta_lake_table
+```
+
+### Delta Lake + S3
+
+```yaml
+  - from: delta_lake:s3://my_bucket/path/to/s3/delta/table/  # A reference to a table in S3
+    name: my_delta_lake_table
+    params:
+      delta_lake_aws_region: us-west-2 # Optional
+      delta_lake_aws_access_key_id: ${secrets:aws_access_key_id}
+      delta_lake_aws_secret_access_key: ${secrets:aws_secret_access_key}
+      delta_lake_aws_endpoint: s3.us-west-2.amazonaws.com # Optional
+```
+
+### Delta Lake + Azure Blob
+
+```yaml
+  - from: delta_lake:abfss://my_container@my_account.dfs.core.windows.net/path/to/azure/delta/table/  # A reference to a table in Azure Blob
+    name: my_delta_lake_table
+    params:
+      # Account Name + Key
+      delta_lake_azure_storage_account_name: my_account
+      delta_lake_azure_storage_account_key: ${secrets:my_key}
+
+      # OR Service Principal + Secret
+      delta_lake_azure_storage_client_id: my_client_id
+      delta_lake_azure_storage_client_secret: ${secrets:my_secret}
+
+      # OR SAS Key
+      delta_lake_azure_storage_sas_key: my_sas_key
+```
+
+### Delta Lake + Google Storage
+
+```yaml
+    params:
+      delta_lake_google_service_account_path: /path/to/service-account.json
+```
+
+## Using secrets
 
-#### Google Storage (GCS)
+There are currently three supported [secret stores](/components/secret-stores/index.md):
```

Comment:
Aren't all of the secret stores supported?

Reply:
Fixed